### DISCUSSION TITLE
Scrolling Tutorial [PATCHED v132]

### DISCUSSION THREAD

**Discussion by py660** [3kh0/ext-remover#1411] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
<a href="https://www.youtube.com/watch?v=ijkREFIS6l0"><img width="1707" alt="Screenshot 2024-12-28 at 2 05 16 AM" src="https://github.com/user-attachments/assets/b8c578b6-04ba-45b1-8e3c-45547a47ea0d" /></a>
https://www.youtube.com/watch?v=ijkREFIS6l0

Need help scrolling? We've got you covered. With our 9th edition tutorial, you'll be scrolling like a pro in no time!

<hr/>

Behind this tutorial's deceiving facade, a lot of work and *Cutting Edge Technology* (TM) actually went into this video. For the audio, I performed meticulous buffoonery using Audacity, and continued screwing up the footage, some of which were rendered using Blender, through the free version of DaVinci Resolve.

Comment below if you know of any other tutorials I should do!

**Comment by 88411** [comment]
Ty
I've been needing this

**Reply by 88411** [reply]
Which one shows you how to execute code on read and write?

**Reply by 88411** [reply]
Can you hurry up and tell me before this gets patched?? 😅

**Reply by py660** [reply]
i think @schoolexploitkid might know... timezones, unfortunately, seems to be getting in the way of efficient interregional communication.

**Reply by 88411** [reply]
Ok but wait. Why when we asked in the other post how to execute code on read and write did you give us a scrolling tutorial of it isn't in the repo

**Reply by 88411** [reply]
Is it actually possible?

**Reply by schoolexploitkid** [reply]
?

**Reply by py660** [reply]
i heard you know of a read&write exploit

I also recall someone telling me about something that worked for both read&write and snap&read or something like that

**Reply by 88411** [reply]
Bruh

**Reply by akabutnicer** [reply]
> i heard you know of a read&write exploit
> 
> I also recall someone telling me about something that worked for both read&write and snap&read or something like that

i heard you know of a read&write exploit

I also recall someone telling me about something that worked for both read&write and snap&read or something like that

**Reply by schoolexploitkid** [reply]
ye its same for read and write and snap and read

**Reply by py660** [reply]
ok... @88411 here ya go

**Comment by py660** [comment]
Here is the snap&read  thing that @schoolexploitkid was supposedly referencing:

https://chrose.netlify.app/detail/snap&run/

> You can work around the patch with:
> ...
>     * [Snap&Run](https://chrose.netlify.app/detail/Snap&Run)

**Reply by py660** [reply]
Alternative URL's will be provided if necessary

**Reply by S-PScripts** [reply]
https://ext-remover.net/detail/Snap&Run/

sex

**Reply by py660** [reply]
IDK how this applies to read&write, but ok

**Reply by akabutnicer** [reply]
> https://ext-remover.net/detail/Snap&Run/
> 
> sex

pay gorn 🤫

**Reply by py660** [reply]
x'es

**Reply by RacoonyCoon** [reply]
> Here is the snap&read thing that @schoolexploitkid was supposedly referencing:
> 
> https://chrose.netlify.app/detail/snap&run/
> 
> > You can work around the patch with:
> > ...
> > * [Snap&Run](https://chrose.netlify.app/detail/Snap&Run)

I see how their are similiartys but S-PScripts said I we can use read&write on my post https://github.com/3kh0/ext-remover/discussions/1404#discussioncomment-11763266

**Comment by 88411** [comment]
It got patched. Can you make a new method?

**Reply by AshtonDavies** [reply]
No, unfortunately you cannot scroll anymore.

**Comment by Charlieis1nsan3** [comment]
wtf is scrolling

**Comment by Charlieis1nsan3** [comment]
OMG what is scrolling 🤪
